### DISCUSSION TITLE
Scrollable call tips

### DISCUSSION THREAD

**Discussion by merlindiavova** [orbitalquark/textadept#427] [Q&A]
If there a way to enable scrolling on call tips. I'm using V12 and when the LSP brings up the call tip for `string.format` my entire buffer gets hidden behind the call tip.

Is there a way to control the dimensions and add scrolling to call tips?

Thanks.

**Comment by orbitalquark** [comment]
I have run into this too. Unfortunately, Scintilla (Textadept's editing component) does not support this feature :(
